docker-compose.yml

```diff
@@ -57,7 +40,7 @@ services:
       - HUB_DB_ENV_MYSQL_USER=hubdb_user
       - HUB_DB_ENV_MYSQL_PASSWORD
       - HUB_DB_PORT_3306_TCP_ADDR=hub-db
-      - HUB_EXT_FILE_URL=http://localhost:8181/sites/default/files
+      - HUB_EXT_FILE_URL=http://localhost:11001/sites/default/files
```

Comment:
This doesn't appear to be relevant to the rest of the PR, was this intentional and can you confirm you're happy with it?

Reply:
I'm happy with this to go in, it's just a fix I threw in there whilst I was updating the docker file. From my understanding the URL refers to the BE service, so I've updated the port to what is being forwarded on line 49.